apps/etterlatte-saksbehandling-ui/client/src/components/person/sakOgBehandling/SakOversikt.tsx

```diff
@@ -112,3 +104,7 @@ const SakHeaderWrapper = styled.div`
   height: 100vh;
   min-width: 25rem;
 `
+
+const OpprettRevurderingWrapper = styled.div`
+  margin-top: 1rem;
```

Comment:
Er denne nødvendig? Ser du har lagt den i `SpaceChildren` som jeg antok fikset dette?

Reply:
`SpaceChildren` har i dag ingen form for spacing, avventer å fjerne denne wrapperen nå til det større arbeidet med å migrere over til Aksel sine `VStack` og `HStack`